docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,192 @@
+
+In [DDD](../guide/ddd.md), the input quantum circuit is modified by inserting gate sequences at regular intervals designed to reduce interaction between (i.e., decouple) the qubits from their environment. 
+
+In [ZNE](../guide/zne.md), the expectation value of the observable of interest is computed at different noise levels, and subsequently the ideal expectation value is inferred by extrapolating the measured results to the zero-noise
+limit. 
+
+
+## Setup
+
+We begin by importing the relevant modules and libraries required for the rest of this tutorial.
+
+```python
+import cirq
+import numpy as np
+from mitiq import MeasurementResult, Observable, PauliString
+```
+
+## Task
+
+We will demonstrate quantum error mitigation on a [GHZ state](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state), entangling multiple qubits together. We can create a short function to do this for convenience, including an optional argument `idle_steps` which will give us a way to insert additional time steps in which to let our qubits idle. This is to more clearly simulate the effect of time correlated noise.
+
+```python
+def ghz(num_qubits, idle_steps=0):
+    # Create  qubit registers
+    qubits = cirq.LineQubit.range(num_qubits)
+
+    # Create a quantum circuit
+    circuit = cirq.Circuit()
+    # Add a Hadamard gate to the first qubit
+    circuit.append(cirq.H(qubits[0]))
+
+    # Add CNOT gates to entangle the first qubit with each of the other qubits
+    for i in range(1, num_qubits):
+        circuit.append(cirq.CNOT(qubits[0], qubits[i]))
+        # Set qubits to idle for specfied number of steps
+        # in the form of Identity gates
+        for step in range(idle_steps):
+            circuit.append(cirq.I(q) for q in qubits)
+
+    return circuit
+```
+
+```python
+num_qubits = 6
+circuit = ghz(num_qubits, 2)
+print(circuit)
+```
+
+## Noise model and executor
+
+**Importantly**, since DDD is designed to mitigate time-correlated (non-Markovian) noise, we simulate systematic $R_z$ rotations and depolarising noise applied to each qubit after each time step. This corresponds to noise which is strongly time-correlated and, therefore, likely to be mitigated by DDD.
+
+We use an [executor function](../guide/executors.md) to run the quantum circuit with the noise model applied.
+
+```python
+def execute(
+    circuit: cirq.Circuit, 
+    rz_noise: float = 0.01,
+    depolar_noise: float = 0.0005
+    ) -> MeasurementResult:
+    """
+    Execute a circuit with R_z dephasing noise of strength ``rz_noise`` and depolarizing noise ``depolar_noise``
+    """
+    # Simulate systematic dephasing (coherent RZ) on each qubit for each moment.
+    circuit = circuit.with_noise(cirq.rz(rz_noise))
+
+    # Simulate systematic depolarizing on each qubit for each moment.
+    circuit = circuit.with_noise(cirq.bit_flip(depolar_noise))
+
+    # Measure out all qubits
```

Comment:
Usually I am not a fan of a comment for each line of code like in this function, but I think for tutorials it works great 😊